server/clcachesrv.py

```diff
@@ -114,9 +127,24 @@ def onSigterm(handle, signum):
 def main():
     logging.basicConfig(format='%(asctime)s [%(levelname)s]: %(message)s', level=logging.INFO)
 
+    parser = argparse.ArgumentParser(description='Process some integers.')
+    parser.add_argument('--exclude', metavar='REGEX', action='append', \
+                        help='Regex ( re.search() ) for exluding of directory watching. Can be specified multiple times. \
+                              Pathnames to test against are after os.path.normcase(), i.e. all lowercase with backslashes. \
```

Comment:
I think there's no pressing need to mention the implementation detail of `os.path.normcase` being used in the help output. How about omitting the entire second sentence and just relying on it working as expected? Since this is on Windows, I'd certainly expect the paths to be case-insensitive and backslashes being used as the separators.